js/cssEditor.js

```diff
@@ -0,0 +1,10 @@
+function starter(){
+    //My variables selecting the necessary text boxes
+let htmlText = document.getElementById("htmlText");
+let cssText = document.getElementById("cssText");
+let browserF = document.getElementById("browserFrame");
+let bfcw = browserF.contentWindow.document;
+
+bfcw.head.innerHTML = '<style type = "text/css">'+ cssText.value + '</style';
```

Comment:
can be cleaner:  `<style type = "text/css">${cssText.value}</style>`

Reply:
Ahh thank you for your input, I'll be sure to clean it up.